dev_guide/managing_images.adoc

```diff
@@ -477,8 +477,9 @@ $ oc secrets new-dockercfg <pull_secret_name> \
     --docker-password=<password> --docker-email=<email>
 ----
 
-To use a secret for pulling images for pods, you must add the secret to your
-service account. The name of the service account in this example should match
+If `serviceAccountConfig.limitSecretReferences` is set to `true` in the master
```

Comment:
@jim-minter Sorry, maybe I'm missing something. Is that not what you meant in the BZ:

"https://docs.openshift.com/container-platform/3.3/dev_guide/managing_images.html / Allowing Pods to Reference Images from Other Secured Registries
- the advice about 'oc secrets link' is superfluous unless serviceAccountConfig.limitSecretReferences is on, which it is probably not in the majority of installs."

Am I missing something obvious?